.github/workflows/deploy.yml

```diff
@@ -80,15 +82,15 @@ jobs:
           target: ${{ matrix.target }}
 
       - name: Setup | musl tools
-        if: matrix.target == 'x86_64-unknown-linux-musl'
+        if: endsWith( matrix.target, 'musl' )
         run: sudo apt install -y musl-tools
 
       - name: Build | Build
-        if: matrix.target != 'x86_64-unknown-linux-musl'
+        if: endsWith( matrix.target, 'musl' )
```

Comment:
Isn’t this supposed to be negated?  As in, `not endsWith(matrix.target, ‘musl’)`, not sure about the syntax though.